README.md

```diff
@@ -1,74 +1,81 @@
-Compressing objects: 100% (153/153), done.
-Writing objects: 100% (224/224), 368.18 KiB | 0 bytes/s, done.
-Total 224 (delta 61), reused 224 (delta 61)
+remote available at ssh://git@deis-builder.deis.rocks:2222/phptest.git
+$ deis config:set BUILDPACK_URL=https://github.com/heroku/heroku-buildpack-php
+$ git push deis
+Counting objects: 247, done.
+Delta compression using up to 8 threads.
+Compressing objects: 100% (176/176), done.
+Writing objects: 100% (247/247), 376.76 KiB | 1.32 MiB/s, done.
+Total 247 (delta 61), reused 247 (delta 61)
+remote: Resolving deltas: 100% (61/61), done.
 Starting build... but first, coffee!
+-----> Restoring cache...
+       No cache file found. If this is the first deploy, it will be created now.
+-----> Fetching custom buildpack
 -----> PHP app detected
 -----> Bootstrapping...
 -----> Installing platform packages...
-       - php (7.0.9)
-       - apache (2.4.20)
+       - php (7.3.1)
+       - apache (2.4.37)
        - nginx (1.8.1)
 -----> Installing dependencies...
-       Composer version 1.1.3 2016-06-26 15:42:08
+       Composer version 1.8.0 2018-12-03 10:31:16
        Loading composer repositories with package information
        Installing dependencies from lock file
-       - Installing psr/http-message (1.0.1)
-       Downloading: 100%
-
-       - Installing pimple/pimple (v3.0.2)
-       Downloading: 100%
-
-       - Installing nikic/fast-route (v1.0.1)
-       Downloading: 100%
-
-       - Installing container-interop/container-interop (1.1.0)
-       Downloading: 100%
-
-       - Installing slim/slim (3.5.0)
-       Downloading: 100%
-
+       Package operations: 5 installs, 0 updates, 0 removals
+         - Installing psr/http-message (1.0.1): Downloading (100%)
+         - Installing pimple/pimple (v3.0.2): Downloading (100%)
+         - Installing nikic/fast-route (v1.0.1): Downloading (100%)
+         - Installing container-interop/container-interop (1.1.0): Downloading (100%)
+         - Installing slim/slim (3.5.0): Downloading (100%)
        Generating optimized autoload files
 -----> Preparing runtime environment...
 -----> Checking for additional extensions to install...
 -----> Discovering process types
        Procfile declares types -> web
        Default process types for PHP -> web
------> Compiled slug size is 14M
-Build complete.
+-----> Checking for changes inside the cache directory...
+       Files inside cache folder changed, uploading new cache...
+       Done: Uploaded cache (232K)
+-----> Compiled slug size is 16M
+Build complete.->
 Launching App...
-Done, zanier-zeppelin:v2 deployed to Deis
+...
+...
+...
+Done, phptest:v3 deployed to Workflow
 
 Use 'deis open' to view this application in your browser
 
 To learn more, use 'deis help' or visit https://deis.com/
 
-To ssh://git@deis-builder.deis.rocks:2222/zanier-zeppelin.git
+To ssh://deis-builder.deis.rocks:2222/phptest.git
  * [new branch]      master -> master
-$ curl http://zanier-zeppelin.deis.rocks
```

Comment:
Could you put these lines back so the documentation example shows the example working?

Reply:
Sure, my bad!